docs/source/schemas/index.rst

```diff
@@ -118,48 +118,56 @@ buffer GeneratedMessages, and palantir Persistables.
-
-Note that all long data types are signed, but VAR\_SIGNED\_LONG is
-encoded in a manner which stores negative numbers more efficiently than
-VAR\_LONG. (The latter will always use 10 bytes to encode negative
-values, but the former will not.)
+characteristics, among other things. The supported types are:
+
++-------------------------------+--------------+---------------+----------------+----------+
+| Name                          | Java Type    | Format        | Anywhere in    | Range    |
+|                               |              |               | table?         | Scans?   |
++===============================+==============+===============+================+==========+
+| FIXED\_LONG                   | long         | byte[8]       | YES            | YES      |
++-------------------------------+--------------+---------------+----------------+----------+
+| FIXED\_LONG\_LITTLE\_ENDIAN   | long         | byte[8]       | YES            | NO       |
+| [3]_                          |              |               |                |          |
++-------------------------------+--------------+---------------+----------------+----------+
+| NULLABLE\_FIXED\_LONG         | long         | byte[9]       | YES            | YES      |
++-------------------------------+--------------+---------------+----------------+----------+
+| VAR\_LONG                     | long         | byte[len] [1]_| YES            | YES      |
++-------------------------------+--------------+---------------+----------------+----------+
+| VAR\_SIGNED\_LONG             | long         | byte[len] [1]_| YES            | YES      |
++-------------------------------+--------------+---------------+----------------+----------+
+| UUID                          | UUID         | byte[16]      | YES            | YES      |
++-------------------------------+--------------+---------------+----------------+----------+
+| SHA\_256\_HASH                | Sha256Hash   | byte[32]      | YES            | YES      |
++-------------------------------+--------------+---------------+----------------+----------+
+| STRING                        | String       | byte[]        | NO [2]_        | YES      |
++-------------------------------+--------------+---------------+----------------+----------+
+| VAR\_STRING                   | String       | byte[len]     | YES            | NO       |
++-------------------------------+--------------+---------------+----------------+----------+
+| BLOB                          | byte[]       | byte[]        | NO [2]_        | YES      |
++-------------------------------+--------------+---------------+----------------+----------+
+| SIZED\_BLOB                   | byte[]       | byte[len]     | YES            | NO       |
++-------------------------------+--------------+---------------+----------------+----------+
+
+.. [1]
+  All long data types are signed, but VAR\_SIGNED\_LONG is
+  encoded in a manner which stores negative numbers more efficiently than
```

Comment:
Prefer to leave the wording as is, because this is true even for negative numbers with large magnitude. For example, storing Integer.MIN_INT could take up 4 bytes instead of 10 (even though Integer.MIN_INT is not "close to zero").